mk/containerized.mk

```diff
@@ -19,7 +19,12 @@ pull: ## Download SVC images
 build:  ## Build SVC images
 build: DOCKER_COMPOSE_ENV = DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1
 
-push: ## Push SVC container images to a registry. Requires previous "docker login"
+GIT_LATEST_TAG != git describe --tags --abbrev=0
```

Comment:
I still think that we should defer this fix/improvement to a later PR. The `nvmeof` and `nvmeof-cli` image tags are defined from the `.env` file (`NVMEOF_VERSION` and `NVMEOF_CLI_VERSION`), so this will result into pushing a tag that is not later used by the rest of the `make`/`docker-compose` commands... And that will be confusing.

Reply:
The `NVMEOF_VERSION` parameter is dynamic now (not taken from .env file as hardcoded parameter)